snuba/admin/views.py

```diff
@@ -228,6 +229,26 @@ def reverse_migration(group: str, migration_id: str) -> Response:
     )
 
 
+@application.route(
+    "/migrations/<group>/overwrite/<migration_id>/status/<new_status>",
+    methods=["POST"],
+)
+@check_tool_perms(tools=[AdminTools.MIGRATIONS])
```

Comment:
is this as restrictive a permission as we can get? because this seems more dangerous than just being able to run predefined scripts.

Reply:
ugh so boring